Integrations/integration-Netskope.yml

```diff
@@ -0,0 +1,307 @@
+                if (args.timeperiod == null){
+                    result = sendRequest('GET', "https://" + url + "/api/v1/alerts?token=" + token + "&type=" + encodeURIComponent(args.type) + "&starttime=" + arg.starttime + "&endtime=" + args.endtime);
+                } else {
+                result = sendRequest('GET', "https://" + url + "/api/v1/alerts?token=" + token + "&type=" + encodeURIComponent(args.type) + "&timeperiod=" + args.timeperiod);
+                }
+            } else {
+                result = sendRequest('GET', "https://" + url + "/api/v1/alerts?token=" + token + "&type=" + encodeURIComponent(args.type) + "&timeperiod=" + args.timeperiod);
+            }
+            var keys = ["App","Timestamp","DLP_Profile","DLP_File","Hostname","Policy"];
+            var retArray = [];
+            result.obj.data.forEach( function (arrayItem)
+            {
+                var app =  arrayItem.app;
+                var timestamp =  arrayItem.timestamp;
+                var dlp_profile =  arrayItem.dlp_profile;
+                var dlp_file =  arrayItem.dlp_file;
+                var hostname =  arrayItem.hostname;
+                var policy =  arrayItem.policy;
+                retArray.push({"App" : app, "Timestamp" : timestamp, "DLP_Profile" : dlp_profile, "DLP_File" : dlp_file, "Hostname" : hostname, "Policy" : policy})
+            });
+            var hr = tblToMd('Events Table', retArray, keys);
+            return {
+               ContentsFormat: formats.table,
+               Type: entryTypes.note,
+               HumanReadable: hr,
+               EntryContext: {
+                    "Netskope.Events.app" : retArray.App,
+                    "Netskope.Events.timestamp" : retArray.Timestamp,
+                    "Netskope.Events.activity" : retArray.Activity,
+                    "Netskope.Events.object" : retArray.Object,
+                    "Netskope.Events.hostname" : retArray.Hostname,
+                    "Netskope.Events.Category" : retArray.Category,
+                    "Netskope.Events.device_classification" : retArray.Device_Classification,
+                    "Netskope.Events.user" : retArray.User,
+                    "Netskope.Events.from_user" : retArray.From_User,
+                    "Netskope.Events.to_user" : retArray.To_User,
+                    "Netskope.Events.srcip" : retArray.SourceIP,
+                    "Netskope.Events.access_method" : retArray.Access_Method,
+                    "Netskope.Events.url" : retArray.URL
+               }
+            }
+    }
+  type: javascript
+  commands:
+  - name: NetskopeEvents
+    arguments:
+    - name: query
+      description: filter query (e.g. user eq jtesarz@netskope.com)
+    - name: timeperiod
+      auto: PREDEFINED
+      predefined:
+      - "3600"
+      - "86400"
+      - "604800"
+      - "2592000"
+      - "5184000"
+      - "7776000"
+      description: TimePeriod
+    - name: starttime
+      description: start time in epoch
```

Comment:
It would be better user experience if the user will pass the date in human readable format and the integration would convert the date to epoch time and send to netskope.